java/src/server/main/Config.java

```diff
@@ -12,4 +12,5 @@
     public static IFacade facade = ServerFacade.getInstance();
     public static String host = "localhost";
     public static int port = 8081;
+    public static String persistenceLoc = "";
```

Comment:
The Config should hold a reference to an IDatabase, not an arbitrary String. In the main class, the IDatabase will be configured based on what was passed on the command line.